incubator/neo4j/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for Neo4j.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+name: "neo4j"
+
+# Specs for the Neo4j docker image
+image: "neo4j"
+imageTag: "3.2.2-enterprise"
+imagePullPolicy: "IfNotPresent"
+
+# Specs for the images used for running tests against the Helm package
+testImage: "markhneedham/k8s-kubectl"
+testImageTag: "master"
+
+# Ports
+boltPort: 7687
+httpPort: 7474
+txPort: 6000
+
+# Cores
+core:
+  numberOfServers: 3
+  discoveryPort: 5000
+  raftPort: 7000
+
+  persistentVolume:
+    ## core server data Persistent Volume Claim annotations
+    ##
+    annotations: {}
+
+    ## core server data Persistent Volume mount root path
+    ##
+    mountPath: /data
+
+    ## core server data Persistent Volume size
+    ##
+    size: 10Gi
+
+    ## core server data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    storageClass: "standard"
+
+    ## Subdirectory of core server data Persistent Volume to mount
+    ## Useful if the volume's root directory is not empty
+    ##
+    subPath: ""
+
+# Read Replicas
+readReplica:
+  numberOfServers: 0
```

Comment:
You'd usually only add read replicas if your read load was high enough to warrant it - you don't necessarily need to have any configured by default.